lib/src/bazelify/build.dart

```diff
@@ -282,14 +289,33 @@ class DartWebApplication implements DartBuildRule {
         '    script_file = "$scriptFile",\n'
         '    deps = _PUB_DEPS';
     if (includeLibraries.isEmpty) {
-      return '$buffer,\n)';
+      buffer = '$buffer,\n)';
     } else {
-      return buffer +
-          ' + [\n' +
+      buffer += ' + [\n' +
           includeLibraries
               .map/*<String>*/((l) => '        ":${l.name}",\n')
               .join() +
           '    ],\n)';
     }
+    buffer += '\ndev_server(\n'
+        '    name = "${name}_dartium_serve",\n'
+        '    deps = _PUB_DEPS + [\n' +
+        includeLibraries.map((l) => '        ":${l.name}",\n').join() +
+        '    ],\n'
+        '    data = glob(["web/**"]),\n'
+        ')';
+    buffer += '\ndart_ddc_bundle(\n'
+        '    name = "${name}_ddc_bundle",\n'
+        '    entry_library = "$scriptFile",\n'
+        '    entry_module = ":$package",\n'
+        '    input_html = "web/index.html",\n'
+        '    output_dir = "web",\n'
+        ')';
+    buffer += '\ndev_server(\n'
+        '    name = "${name}_ddc_serve",\n'
+        '    deps = [":${name}_ddc_bundle"],\n'
```

Comment:
All you should need here is a `name` and `data = [":${name}_ddc_bundle"]`. The bundle rule includes in its runfiles all the transitive dart sources, sourcemaps, etc (this was fixed recently).